project/jmh.scala

```diff
@@ -8,58 +9,29 @@ object RenaissanceJmh {
 
   def generateJmhWrapperBenchmarkClass(info: BenchmarkInfo, outputDir: File): File = {
     val packageName = info.benchClass.getPackage.getName
-    val name = info.benchClass.getSimpleName
-    val content = s"""
-     package $packageName;
-
-     import java.util.concurrent.TimeUnit;
-     import static java.util.concurrent.TimeUnit.MILLISECONDS;
-     import org.openjdk.jmh.annotations.*;
-     import org.renaissance.RenaissanceBenchmark;
-     import org.renaissance.JmhRenaissanceBenchmark;
+    val jmhClassName = "Jmh" + info.benchClass.getSimpleName
 
-     @State(Scope.Benchmark)
-     @OutputTimeUnit(TimeUnit.MILLISECONDS)
-     @Warmup(iterations = ${info.repetitions}, time = 1, timeUnit = MILLISECONDS)
-     @Measurement(iterations = ${info.repetitions / 4 + 1}, time = 1, timeUnit = MILLISECONDS)
-     public class Jmh_$name extends JmhRenaissanceBenchmark {
-       public String benchmarkName() {
-           return RenaissanceBenchmark.kebabCase("$name");
-       }
-
-       @Setup(Level.Trial)
-       public void setUpBeforeAll() {
-         defaultSetUpBeforeAll();
-       }
+    val content = s"""
+package ${packageName};
 
-       @Setup(Level.Iteration)
-       public void setUp() {
-         defaultSetUp();
-       }
+import org.openjdk.jmh.annotations.*;
+import org.renaissance.jmh.JmhRenaissanceBenchmark;
 
-       @TearDown(Level.Iteration)
-       public void tearDown() {
-         defaultTearDown();
-       }
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
 
-       @TearDown(Level.Trial)
-       public void tearDownAfterAll() {
-         defaultTearDownAfterAll();
-       }
+@State(Scope.Benchmark)
+@OutputTimeUnit(MILLISECONDS)
+@Warmup(iterations = ${info.repetitions}, time = 1, timeUnit = MILLISECONDS)
+@Measurement(iterations = ${info.repetitions / 4 + 1}, time = 1, timeUnit = MILLISECONDS)
+public class ${jmhClassName} extends JmhRenaissanceBenchmark {
+  public ${jmhClassName}() { super("${info.name}"); }
+}
+"""
```

Comment:
Is `scalafmt` ok with this indentation?

Reply:
As far as I can tell, it is :-) I don't actually see what's wrong with it when it is in a string.